.github/workflows/mirror_community_pipeline.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           else
             # e.g. refs/tags/v0.28.1 -> v0.28.1
             echo "CHECKOUT_REF=${{ github.ref }}" >> $GITHUB_ENV
-            echo "PATH_IN_REPO=${${{ github.ref }}#refs/tags/}" >> $GITHUB_ENV
+            echo "PATH_IN_REPO=$(echo ${{ github.ref }} | sed 's/^refs\/tags\///')" >> $GITHUB_ENV
```

Comment:
Tricky one. Thanks for taking care of this. 

(side note: having an automated reporting mechanism would have promptly alerted us about this; so all the reasons for me to work on it soon)

Reply:
Definitely!

Even more subtle, the github action passed (:heavy_check_mark:) but its behavior was wrong. When working on a reporting mechanism, it would be good to report values from `PATH_IN_REPO`/`CHECKOUT_REF` to slack as well.